e2e/package.json

```diff
@@ -25,8 +25,8 @@
     "test-local-devup:debug": "cross-env WORKBENCH_ENV=local PUPPETEER_HEADLESS=false DEBUG=true node --inspect-brk node_modules/.bin/jest --maxWorkers=1",
     "test-staging": "cross-env WORKBENCH_ENV=staging yarn _test --maxWorkers=3",
     "test-staging:debug": "cross-env WORKBENCH_ENV=staging PUPPETEER_HEADLESS=false DEBUG=true yarn _test --maxWorkers=1",
-    "test:ci": "yarn _test --ci --maxWorkers=2 --forceExit",
-    "test:ci:debug": "yarn _test --ci --debug --maxWorkers=1 --bail 1",
+    "test:ci": "yarn _test --ci --maxWorkers=3 --forceExit",
+    "test:ci:debug": "yarn _test --ci --debug --maxWorkers=3",
```

Comment:
Why change the bail setting here?

Reply:
I think we still want to run all test suites even if there is one failed suite. I don't feel strongly. Should I add it back?

Reply:
Sounds fine, I was just curious why it was changing as part of this PR.